ush/forecast_postdet.sh

```diff
@@ -515,6 +515,8 @@ FV3_out() {
   else
     # No need to copy FV3 restart files when RUN=gfs or gefs
     ${NCP} "${DATA}/input.nml" "${COM_ATMOS_HISTORY}/input.nml"
+    ${NCP} "${DATA}/model_configure" "${COM_ATMOS_HISTORY}/model_configure"
+    ${NCP} "${DATA}/nems.configure" "${COM_ATMOS_HISTORY}/nems.configure"
```

Comment:
I think we now have `COM_CONF`.  Please copy these files (and `input.nml`) to `COM_CONF` with a prefix `ufs.`

Reply:
So these lines would become: 

${NCP} "${DATA}/model_configure" "${COM_CONF}/ufs.model_configure"
${NCP} "${DATA}/nems.configure" "${COM_CONF}/ufs.nems.configure"

Correct?

Reply:
I tried the lines above but the test failed with the error:
/scratch1/BMC/wrfruc/mtoy/git_local/global-workflow.test/ush/forecast_postdet.sh: line 517: COM_CONF: unbound variable

@aerorahul Can you clarify what needs to be done here?  Thank you.

Reply:
Can I get some more specific details on which scripts need to be modified and what needs to be added?  I tried this with jobs/JGLOBAL_FORECAST:
@@ -46,7 +46,7 @@ declare -rx gcyc="${GDATE:8:2}"
 
 # Construct COM variables from templates (see config.com)
 YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_RESTART COM_ATMOS_INPUT COM_ATMOS_ANALYSIS \
-  COM_ATMOS_HISTORY COM_ATMOS_MASTER COM_TOP
+  COM_ATMOS_HISTORY COM_ATMOS_MASTER COM_TOP COM_CONF
 
 RUN=${rCDUMP} YMD="${gPDY}" HH="${gcyc}" generate_com -rx \
   COM_ATMOS_RESTART_PREV:COM_ATMOS_RESTART_TMPL

But it did not work.

Thank you.

Reply:
That should've been enough. Can you point me to your log?

Reply:
Just chatted with @aerorahul I think we also need to create the folder and he showed me where.  I'll try to add this after my current meeting.

Reply:
@WalterKolczynski-NOAA  The log file is on Hera at:
/scratch1/BMC/wrfruc/mtoy/git_local/global-workflow.test/FV3GFSrun/GFS_v17_test.C768/logs/2019110300/gfsfcst.log
And my modified JGLOBAL_FORECAST file is at:
/scratch1/BMC/wrfruc/mtoy/git_local/global-workflow.test/jobs/JGLOBAL_FORECAST

Reply:
@JessicaMeixner-NOAA Is the folder you discussed creating a 'conf' folder?  The input.nml, etc. files would be sent there instead of the 'history' directory?  I kind of liked having them go to the 'history' directory.  :)

Reply:
@mdtoyNOAA apologies, I didn't get to finish this testing before leaving for the weekend.  Since configuration files are not considered model history files, these files will all go to the "conf" directory where other configuration files will also go.  Yes, the CONF directory is the directory folder where things should go.  I'll start looking at this again now.

Reply:
@JessicaMeixner-NOAA  Sounds good.  Thank you for the update.